### PR TITLE
Updated CHANGELOG and metadata for upcoming 1.4.0 release.

### PR DESCRIPTION
**Pull request by jonnytpuppet**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,21 @@
...
+- Added 'ssl_certificate_id' property to elb_loadbalancer.
+
+As well as following fixes and improvements:
+- Fixed multiple issues related to the region property being displayed/returned incorrectly.
+- Documentation improvements.
+- Multiple test improvements.
+- Rubocop updates.
+- Fixed issue with elb_loadbalancer availability_zones synchronisation detection.
+- Use the VPC's default subnet when none is specified on the ec2_instance.
+- Use the region as part of the cache key in the base provider.
```
Comment (DavidS) [inline CHANGELOG.md]: The important bit here is "Enable `puppet resource` usage across regions for ec2_vpc (and maybe others)."

**Comment by DavidS** [comment]
Please add Debian 8 support in the metadata, and mention https://github.com/puppetlabs/puppetlabs-aws/commit/90813639991fdacbcdc40f44b85ad73952a0dd3b .

**Comment by jonnytpuppet** [comment]
I tried to mention that change as part of "Fixed multiple issues related to the region property being displayed/returned incorrectly."

**Comment by jonnytpuppet** [comment]
@DavidS Code updated

**Comment by DavidS** [comment]
The parse_ini/ini_parse issue is a hard crash, encountered by anyone using the config file.

**Pull request merged**